core/app/models/workarea/data_file/json.rb

```diff
@@ -45,12 +45,55 @@ def find_updated_model_for(attrs)
 
         if id.present?
           result = model_class.find_or_initialize_by(id: id)
-          result.attributes = attributes
+          result.attributes = attributes_without_updated_at(attributes)
+          result.touch if result.changed?
```

Comment:
I'm not sure if this is a Mongoid bug, but the reason I had to go ham on the recursion with `#touch` was because when you mass-assign attributes like this it actually doesn't seem to touch the embedded models...

Reply:
`touch` triggers callbacks and db persistence iirc, we won't want to do that.